view/template/trial.hbs

```diff
@@ -69,6 +69,16 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                    <input type="hidden" id="trialId" name="trialId" value="{{trial.id}}">
```

Comment:
Yes I am preventing default and sending a custom Ajax request but the ajax request is sending all the fields (which includes the hidden field). That's the reason I put it outside the form.